### ISSUE TITLE
White noise - audio not coming through endpoints correctly

### ISSUE THREAD

**Issue by stephen2222** [eiz/SynchronousAudioRouter#88]
Hi there.

I have a problem in Ableton where I can create everything normally, but when I route the endpoint into a track on Ableton the two channels the audio is coming in on are peaking and only letting through white noise.

When I mute the programme (Spotify, YouTube) it mutes the audio coming through so it appears it is sending somewhat correctly. When I look at the windows sound manager, I can see the audio being sent correctly with that SARS audio output not peaking in windows sounds.

Help!

**Comment by MeMaMoo** [comment]
I'm also having this issue.

I'm assuming I've set something up wrong, but I have no idea what. 

Does anyone have any ideas / can point me in the right direction?

I'm using Reaper, and trying to set it up for my XLR mic being captured via Reaper to go to Discords Mic Input, and Discords output to come back to reaper.

The output does sound like it's there, but it's behind a massive wall of white noise.

Please let me know if/what information would be helpful.

**Comment by Christin-Michelle** [comment]
I have the same problem with studio one 5 anyone have any idea? Tried already Different settings in the system tab 44khz 48khz 16 24 32 bit and so on but it doesn't make any difference.

**Comment by Christin-Michelle** [comment]
Update I really tried a lot now. I own a Tascam Model 12 which is my hardware for input and output and provides the asio driver. I usually work with Studio One 5 and want to use SAR to to route JamKazam in and out of Studio ONE 5. 
Now I tried obviously the SAR ASIO in Studio One and  Stereo In-/ and Outputs for JamKazam. It seems the way from ASIO to JaKaz SAR does carry some audio information (taking into accout that the VU meter in JK setup does somehow respond to microfon input). The way back is just white noise(as I can hear). Now I tried changing different settings (sample rate, buffer sizes and what else I could find). I tried setting up JamKazam with ASIO4all (and selecting the in and output channel from SAR as devices in ASIO4ALL) but didn't help any. The funny thing is I was able to set this up using Reaper as DAW (rest of the setup was just alike, also the setup in reaper was as far as I can interpret (I am not familiar with reaper) pretty much alike the setup I use in Studio One. The problem is I only have a demoversion of reaper, I am not familiar with it, actually don't think it has the same benfit for me a S1 has, so changing is not an alternative. 
Basically I think I am about done with what I can do. So if there would be any fix, workaround I would greatly appreaciate btw both program (S1 and reaper) are 64 bit and the operating system also is W10-64.
